D/D.sublime-syntax

```diff
@@ -1465,20 +1463,17 @@ contexts:
   value-condition:
     - match: (?=:)
       pop: 1
```

Comment:
this `pop: 1` should be `pop: true` as the syntax is v1

Reply:
`pop: 1` works in v1 as well and that line wasn't changed in this PR.